modules/platform/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/operations/coldstorage/AbstractTestColdStorageOperation.java

```diff
@@ -66,18 +68,52 @@ protected void moveContentToColdStorage(CoreSession session, DocumentModel docum
         try (OperationContext context = new OperationContext(session)) {
             context.setInput(documentModel);
             DocumentModel updatedDocModel = (DocumentModel) automationService.run(context, MoveToColdStorage.ID);
+            checkMoveContent(List.of(documentModel), List.of(updatedDocModel));
```

Comment:
Worth adding a cleanup commit to make the (old) variables name consistent :
- `documentModel` -> `doc`
- `updatedDocModel` -> `updatedDoc`
- `documents` -> `docs`
- `documentIds` -> `docIds`